watch-package.js

```diff
@@ -57,8 +59,17 @@ module.exports = function watchPackage (pkgDir, exit) {
       return a.concat(b)
     })
 
+    if (ignores) {
+      ignores = [].concat(ignores).map(function (ignore) {
+        return ['--ignore', ignore]
```

Comment:
https://github.com/remy/nodemon#ignoring-files

If a user specifies an `--ignore` value that uses a wildcard, that value must be quoted. Should we require users to quote their ignore values, e.g. `"ignore": "'lib/*.js'"`, or should we work to ensure that all ignore values are quoted here, adding quotes if they don't exist already?